src/cobra/io/sbml.py

```diff
@@ -929,7 +929,7 @@ def write_sbml_model(cobra_model, filename, f_replace=F_REPLACE, **kwargs):
     ----------
     cobra_model : cobra.core.Model
         Model instance which is written to SBML
-    filename : string
+    filename : string or filehandle
```

Comment:
I assume that had nothing to do with the PR? Not really a big change so we can merge it in here as well.

Reply:
Yeah, nothing to do with PR.